### PR TITLE
fix: [DSOPS-106] delete lines 72-74

### PR DESCRIPTION
**Pull request by aiplesca**
### This is a 

- [x] Bug Fix
- [ ] Feature
- [ ] Documentation
- [ ] Other

#### Commit style
   
   - [ ] Changes are on a branch with a descriptive name eg. `fix/missing-queue`, `docs/setup-guide`
   
   - [ ] Commits start with one of `feat:` `fix:` `docs:` `chore:` or similar
   
   - [ ] No excessive commits, eg: there should be no `fix:` commits for bugs that existed only on the PR branch (see [guide-to-interactive-rebasing](https://hackernoon.com/beginners-guide-to-interactive-rebasing-346a3f9c3a6d))

#### Protected files

The following files should not change unless they are directly a part of your change.
    
   - [ ] `yarn.lock` (unless package.json is also modified, then only the new/updated package should be changed here)
   
   - [ ] `package.json` (renovate bot should handle all routine updates)
   
   - [ ] `tsconfig.json` (only make it stricter, making it more lenient requires more discussion)
   
   - [ ] `tslint.json` (only make it stricter, making it more lenient requires more discussion)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ node_js:
 
 branches:
   only:
-    - master
```
Comment (linux-wizard) [inline .travis.yml:10]: You can remove this file entirely as we don't use travis

**Comment by linux-wizard** [comment]
Need to create a new NPM package which will be hosted on yield app GitHub npm registry. You can see an example in the yield-utils repo

File: .travis.yml (hunk 1 above)
Comment (aiplesca) [reply .travis.yml:10]: done

**Review by linux-wizard — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+  "name": "@yieldapp/semantic-release-jira",
   "version": "0.3.2",
   "main": "dist/lib/index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/boxcee/semantic-release-jira.git"
+    "url": "git+https://github.com/yieldapp/semantic-release-jira.git"
   },
   "publishConfig": {
     "access": "public"
```
Comment (linux-wizard) [inline package.json:10]: not sure this should be set to public

**Review by linux-wizard — APPROVED** [review]

**Pull request merged**